client/app/shared/modals/tables/tables.component.html

```diff
@@ -1,35 +1,41 @@
-<div id="tables" class="fast reveal" [ngClass]="{ 'loading': loading }" data-reveal data-animation-in="hinge-in-from-middle-y" data-animation-out="hinge-out-from-middle-y">
-  <h2>Manage tables</h2>
-  <div class="dimmer">
-    <div class="big loader"></div>
+<div class="icon link item" (click)="open()">
+  <i class="ui clipboard icon"></i>
+  <span class="hover text">Manage tables</span>
+</div>
+<div class="ui basic tables-modal small modal" [ngClass]="{ 'loading': loading }">
+  <div class="ui header">
+    Manage tables
   </div>
-
-  <div class="grid-x grid-padding-y">
-      <div *ngIf="tables().length === 0" class="cell grid-x grid-padding-x">
-          Set names for different tournament tables.
-          <br>
-          These names will appear on the scoresheets
-      </div>
-
-    <div *ngFor="let table of tables(); index as index" id="table-{{index}}" class="cell grid-x grid-padding-x">
-      <input class="cell small-10" (keyup)="tableChanged(table)" [(ngModel)]="table.tableName">
-      <div class="cell small-2">
-        <div class="alert clear button" (click)="removeTable(table)">
-          <i class="fa fa-times"></i>
+  <div class="content">
+    <div *ngIf="tables().length === 0">
+      Set names for different tournament tables.
+      <br/>
+      These names will appear on the scoresheets.
+      <br/>
+    </div>
+    <div class="ui form">
+      <div *ngFor="let table of tables(); index as index" id="table-{{index}}" class="field">
+        <div class="ui fluid action input">
+          <input (keyup)="tableChanged(table)" [(ngModel)]="table.tableName">
+          <div class="ui red icon button" (click)="removeTable(table)">
+            <i class="ui trash icon"></i>
+            <div class="hover text">Remove table</div>
+          </div>
         </div>
       </div>
-    </div>
-    <div class="cell grid-x grid-padding-x">
-      <input class="cell small-10" (keyup)="tableChanged(newTable)" [(ngModel)]="newTable.tableName">
+      <div class="cell grid-x grid-padding-x">
```

Comment:
Those are foundation classes

Reply:
Yes... Didn't notice this. Looking for any leftovers now.